model_managers/model_manager.py

```diff
@@ -220,9 +220,9 @@ def create_relationship(self, rel_list: List[List[str]], identifier='label', mat
         """
         Create relationship nodes between two specified classes as defined in rel_list.
         For example:
-            With rel_list = [ ['class1', 'class2', 'example'] ]
+            With rel_list = [ ['class1', 'class2', 'example', 'false'] ]
```

Comment:
@PranjaliParnerkar 
On the tab2neo level I would not like to mandate the optional status of a relationship to be provided.
Could you please implement the function to support both:
- a list of length 3 (without the optional status)
- and a list of length 4 with optional status
?
Would be useful to have unit tests for both scenarios.
Thanks.

Reply:
@PranjaliParnerkar To add to this, some current unit tests are failing with this change as they do not expect an optional key in the relationships dicts.

Reply:
I have made changes to create_relationship() to allow list with and without optional rel and have updated the test cases as well. Please have a look and let me know if changes are required. Thanks!